README.md

```diff
@@ -66,6 +66,7 @@ Some examples require extra dependencies. See each sample's directory for specif
 * [sentry](sentry) - Report errors to Sentry.
 * [worker_specific_task_queues](worker_specific_task_queues) - Use unique task queues to ensure activities run on specific workers.
 * [worker_versioning](worker_versioning) - Use the Worker Versioning feature to more easily version your workflows & other code.
+* [workflow_update](workflow_update) - Send a request to and a response from a client to a workflow execution.
```

Comment:
I think such a simple feature sample may deserve to be with the others in `hello`. I understand there's temporary extra README info while it is experimental (we can add to bottom of hello README), but otherwise I think of this as a basic demonstration of a basic feature like signals, queries, etc.

(granted would love to do #67 and not have those be single files, but that can come separate/later for all hello samples)